firefox-ios/Client/Frontend/Strings.swift

```diff
@@ -3731,6 +3731,21 @@ extension String {
         tableName: nil,
         value: "OK",
         comment: "OK button to dismiss the error prompt.")
+    public static let ScanQRCodeConfirmOpenURLMessage = MZLocalizedString(
+        key: "ScanQRCode.ConfirmOpenURL.Message.v129",
+        tableName: "ScanQRCode",
+        value: "Allow Firefox to open?",
```

Comment:
We should use the placeholder here, and add a comment that the Placeholder is the app name.

Reply:
```suggestion
        value: "Allow %@ to open?",
```

Reply:
> We should use the placeholder here

Good point, should be fixed now. Thanks